fearless/Modules/Staking/Operations/ValidatorOperationFactory/RelaychainValidatorOperationFactory.swift

```diff
@@ -460,7 +548,7 @@ final class RelaychainValidatorOperationFactory {
                     chainFormat: chainFormat
                 )
 
-                let validatorReturn = try calculator
+                let validatorReturn = try? calculator
```

Comment:
it's additional operation just for calculate one of 9 fields for validator entity. in case rewards calculation failed we still should show this validator in the list. so error catching doesn't make sense.